modules/cnf-disabling-interrupt-processing-for-individual-pods.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-disabling-interrupt-processing-for-individual-pods_{context}"]
+= Disabling interrupt processing for CPUs where pinned containers are running
+
+To achieve low latency for workloads, some containers require that the CPUs they are pinned to do not process device interrupts. A pod annotation, `irq-load-balancing.crio.io`, is used to define whether device interrupts are processed or not on the CPUs where the pinned containers are running. When configured, CRI-O disables device interrupts where the pod containers are running.
+
+To disable interrupt processing for CPUs where containers belonging to individual pods are pinned, ensure that `globallyDisableIrqLoadBalancing` is set to `false` in the performance profile. Then, in the pod specification, set the `irq-load-balancing.crio.io` pod annotation to `disable`.
+
+The following pod specification contains this annotation:
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: Pod
+metadata:
+  annotations:
+      irq-load-balancing.crio.io: "disable"
+spec:
+    runtimeClassName: performance-<profile_name>
+...
```

Comment:
"For [snippets or sections of a file](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#signs-and-symbols-in-code-blocks)... for YAML, use ('#…​')" throughout this code block

```suggestion
# ...
```